test/publishing_api_v2_test.rb

```diff
@@ -139,6 +139,75 @@ def content_item_for_content_id(content_id, attrs = {})
         assert_equal "Unprocessable entity", error.error_details["error"]["message"]
       end
     end
+
+    describe "optimistic locking" do
+      describe "if the content item has not change since it was requested" do
+        before do
+          @content_item = content_item_for_content_id(@content_id, "previous_version" => 3)
+
+          publishing_api
+            .given("the content item is at version 3")
```

Comment:
The given will need to include a specific content_id, which will need to be specified in the paths below.  When setting up the provider state in the producer, there's no way for it to access `@content_id`.  Including it in the state description keeps this obvious, and is also consistent with some of the states below.